test-utils/src/lib.rs

```diff
@@ -141,6 +141,27 @@ pub fn key_pair_from_string(value: impl AsRef<str>) -> ed25519::Pair {
     ed25519::Pair::from_string(format!("//{}", value.as_ref()).as_str(), None).unwrap()
 }
 
+/// Create a key pair derived from the given value and associate
+/// its account id with a User in the registry using the said value,
+/// lowecased, as its id. For accounts that are not 'Alice', a
+/// transfer is done to them to have enough funds to pay for the
+/// tx fee when registering the user.
```

Comment:
The doc is quite complex. How about this?
```suggestion
/// Create a key pair derived from `value` and register a user with that key pair.
/// The user name is `value.to_lowercase()`. Ensures that the account for the key
/// pair is equipped with at least 100.000 RAD.
```

Reply:
Better :ok_hand: thanks!